package.json

```diff
@@ -62,9 +62,7 @@
     "eslint-plugin-import": "^2.28.1",
     "eslint-plugin-n": "^16.1.0",
     "eslint-plugin-promise": "^6.1.1",
-    "front-matter": "^4.0.2",
     "highlight.js": "^11.8.0",
-    "jasmine": "^5.1.0",
```

Comment:
What benefits does node:test providee over Jasmine? Not really a concern, but just curious why the switch.

Reply:
Mostly just so @markedjs/testutils doesn't require extensions to use jasmine. Also I wanted to try the new node test runner.

Reply:
I would flip the question around - what benefits does jasmine provide over the built in test runner? 😄

Because jasmine is 3MB of code https://packagephobia.com/result?p=jasmine